### PR TITLE
Fixed text field clear and link icon

### PR DESCRIPTION
**Pull request by PoplarToppler**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

* **Fixed** text input field clear. It now appears as expected
* **Fixed** primary link to use the new icon 'open-tabs'

### CODE REVIEW

**Comment by PoplarToppler** [comment]
Interesting that that test failed now. I'll look into it later 😃

**Comment by dafortin** [comment]
Looks good ! Thanks :)

**Review by dafortin — APPROVED** [review]

**Comment by PoplarToppler** [comment]
Do I have to do something about this pullapprove error? It's saying that 'notmessenger' is not a collaborator of this repo.

**Comment by dafortin** [comment]
We will have to ask someone that has the permissions to check why this use is not considered as a valid one. Maybe @sglanzer. I approved your PR on GitHub so don't worry about pullapprove :)

**Comment by dafortin** [comment]
Just make sure that the tests are passing before merging

**Comment by PoplarToppler** [comment]
Ok. Yes all the tests are passing now. Do I merge it myself?

**Comment by dafortin** [comment]
How did you got the tests to pass? Intermittent failures?

**Comment by dafortin** [comment]
You can merge it or I can do it has you wish :)

File: tests/integration/components/frost-link-test.js
Comment (PoplarToppler) [inline tests/integration/components/frost-link-test.js:217]: I fixed it here

**Comment by PoplarToppler** [comment]
Alright I'll let you merge it, thanks! :)

**Pull request merged**
